src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -327,11 +326,12 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 	readonly onDidRunText = this._onDidRunText.event;
 	private readonly _onDidChangeTarget = this._register(new Emitter<TerminalLocation | undefined>());
 	readonly onDidChangeTarget = this._onDidChangeTarget.event;
+	private readonly _onDidSendText = this._register(new Emitter<string>());
+	readonly onDidSendText = this._onDidSendText.event;
```

Comment:
We can combine this with onDidInputData, I'll handle this in https://github.com/microsoft/vscode/issues/200242